app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/MapViewModel.kt

```diff
@@ -58,6 +60,7 @@ class MapViewModel constructor(
     fun turnOnTreeCaptureMode() {
         locationDataCapturer.turnOnTreeCaptureMode()
         stepCounter.enable()
+        deviceOrientation.enable()
```

Comment:
Tried to have a function in DeviceOrientation the enables, takesSnapshot and disables the listener to see if that helps to avoid this call here. But that model didn't really register any value for the rotation matrix and hence the need for this call separate from the point where the ImageCaptureActivity calls the takeSnapshotAndDisable method.